library/timing.c

```diff
@@ -230,11 +230,19 @@ unsigned long mbedtls_timing_hardclock( void )
 
     if( hardclock_init == 0 )
     {
+#ifdef __MINGW32__
```

Comment:
The ChangeLog entry says you're also fixing the case of mingw64. According to https://sourceforge.net/p/predef/wiki/Compilers/ `__MINGW64__`  implies `__MINGW32__`, so your code is fine, but if that's the underlying reasoning, I think we should add a comment, or simply add a clause for `__MINGW64__`.

Reply:
added